grpc-server/src/main/java/com/netflix/conductor/grpc/server/service/MetadataServiceImpl.java

```diff
@@ -33,6 +33,15 @@ public MetadataServiceImpl(MetadataService service) {
         this.service = service;
     }
 
+    @Override
+    public void getWorkflows(MetadataServicePb.GetWorkflowsRequest req, StreamObserver<MetadataServicePb.GetWorkflowsResponse > response) {
```

Comment:
**Suggestion**: can we use `getWorkflowDefs` or `getWorkflowDefinitions`?
Workflow objects are instances, while definitions (`WorkflowDef` objects) are within those instances.

Reply:
**Question**: I can see this working with immutable workflows (where the definition is embedded inside). Have you tried running this with a legacy workflow (where the definition is stored on db)?
In addition, If you want to get promoted this to `Netflix:dev` or production on their side, a test might help gain their confidence ;)